tests/vagrant/Vagrantfile

```diff
@@ -6,15 +6,15 @@ VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
-	config.vm.box = "ubuntu/trusty64"
+	config.vm.box = "ubuntu/xenial64"
 	config.vm.box_check_update = false
 
 	config.ssh.forward_agent = true
+	config.vm.network :private_network, ip: '192.168.10.10'
```

Comment:
I keep encountering the following error without it:

NFS requires a host-only network to be created.
Please add a host-only network to the machine (with either DHCP or a
static IP) for NFS to work.